docs/pages/Project.md

```diff
@@ -2,14 +2,14 @@
 
 The [UI5 Project](https://github.com/SAP/ui5-project) module provides functionality to build a UI5 project's dependency tree, including the validated project configurations. It is also responsible for installing and managing framework dependencies. Also see [Development Overview: Project Dependencies](./Overview.md#project-dependencies).
 
-[**API Reference**](https://sap.github.io/ui5-tooling/api/module-@ui5_project.html){: .md-button .sap-icon-initiative }
+[**API Reference**](https://sap.github.io/ui5-tooling/v3/api/@ui5_project_build_ProjectBuilder.html){: .md-button .sap-icon-initiative }
 
 ## Normalizer
-The purpose of the normalizer is to collect dependency information and to enrich it with project configuration (both done in [generateProjectTree](https://sap.github.io/ui5-tooling/api/module-@ui5_project.normalizer.html#.generateProjectTree)).
+The purpose of the normalizer is to collect dependency information and to enrich it with project configuration (both done in [generateProjectTree](https://sap.github.io/ui5-tooling/v3/api/module-@ui5_project.normalizer.html#.generateProjectTree)).
```

Comment:
@RandomByte Could you share some content which part you wanna see in the docs/pages/Project.md?

Reply:
Most of this document is obsolete. Normalizer and Translators got replaced with other things. I think we should rather remove the document for now. But this can be done in a separate PR.

Reply:
I've updated the Project.md by removing all v2 specific content. Whats your opinion on that? IMHO removing the document completely might be lead to a non complete list of modules in the devguide

Reply:
Looks good!